text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
+* Highlight contributions made by community members (plugins built, code PRs)
+* "Official" landing pages to help community members in their personal website marketing efforts
+* Allow companies interested in building Gatsby sites to find individuals nearby them who are open for Gatsby projects
+
+#### Agency Pages
+* Highlight agencies building websites on Gatsby
+* "Official" landing pages to help agencies in their marketing efforts
+* Allow companies interested in building Gatsby sites to find agencies nearby them
+* Help agencies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+#### Company Pages
+* Display list of companies using Gatsby to demonstrate it's "enterprise-ready" and fulfills whatever use-case they're looking for (e-commerce, authentication, etc). 
+* Help companies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+# Detailed design
+
+## Pages
+
+#### All people / agencies / companies
+
+One template with multiple URLs, each of which apply different filters:
+  * /people/ (all people)
+  * /agencies/ (all agencies)
+  * /companies/ (all companies)
+
+Filter buttons should be at the top and should be easy to toggle between them (select-one). Filters should include:
+  * Individual 
+  * Agency 
+  * Company
+  * Open to new contracts (people & agencies with this field toggled)
+  * Now hiring (agencies & companies with this field toggled)
+
+#### Person / agency / company page
+
+One template with multiple URL patterns, depending on the Creator type:
+  * /companies/[company-name]
+  * /agencies/[agency-name]
+  * /people/[person-name]
+
+
+### Data Structures
+
+Showcase sites ([listed here](https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml)) can have multiple Creators.
+
+**Creators** can either be an Individual, a Digital Agency, or a Company. . 
+
+Creator has the following fields:
+* an id, in slug format (eg mike-allanson)
+* type -- Individual, a Digital Agency, or a Company. We use the word Organization below to refer to either a Digital Agency and Company.
+* a description of them in the third person “Eg, Mike Allanson is a developer living in London"
+* Personal website
+* GH handle (?)
+* Open for Work [boolean]
+* A photo / logo (depending on Individual / Organization type)
+* Org they are affiliated with (if Individual)
+* People they are affiliated with (if Org)
+* A list of open jobs (if they are Organization type)
+* A list of all the sites they’ve built.
+* A location with:
+    * A description (“London, UK”)
+    * Lat / lng
+
```

Comment:
What about how to contact them? I guess we rely on their personal site / org site to provide that?

Reply:
This is worth thinking about. I don't know that we want to create a big list of contact info in an open source repo, but we also want it to be easy for a potential client/employer to get in touch.

Reply:
Yep, have this worry too. My thought is to start without it and figure out how to add later. Perhaps `username` and `domain` fields to keep safe from scrapers?